### PR TITLE
setup webpack and delete spockets

### PR DESCRIPTION
**Pull request by marderer**

### CODE REVIEW

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -4,16 +4,14 @@ html
...
-    = stylesheet_pack_tag 'bootstrap'
+    = stylesheet_pack_tag 'style'
     = javascript_pack_tag 'application'
     = stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"
 
   - if user_signed_in?
     body#page-top.fixed-nav.sticky-footer.bg-dark
       nav#mainNav.navbar.navbar-expand-lg.navbar-dark.bg-dark.fixed-top
-        a.navbar-brand href="#{root_url}"  Start Bootstrap
+        a.navbar-brand href="#{root_url}"  Personal finance
```
Comment (AleksSenkou) [inline app/views/layouts/application.html.slim]: locales

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -29,7 +27,7 @@ html
           ul.navbar-nav.ml-auto
             span.balance
               b
-                | Balance:
+                = t('balance') + ':'
```
Comment (AleksSenkou) [inline app/views/layouts/application.html.slim]: @marderer is 
```ruby
= t('balance')
| :
```
workable solution?

adding something to localized text via `+` looks like a hack 😄

File: app/webpack/packs/application.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import 'bootstrap/dist/js/bootstrap';
+import '../src/javascript/sb-admin'
+
+import Rails from 'rails-ujs'
+Rails.start()
+global.Rails = Rails;
+
+import Turbolinks from 'turbolinks'
+Turbolinks.start()
+
```
Comment (AleksSenkou) [inline app/webpack/packs/application.js]: @marderer why did you rename `app/javascript` to `app/webpack`?

Comment (marderer) [reply app/webpack/packs/application.js]: @AleksSenkou because now I want using CSS with webpacker, not only javascript. I think that javascript is a bad name for the folder into which I will add styles and images, not only js.

Comment (AleksSenkou) [reply app/webpack/packs/application.js]: @marderer good notice. got it 👍

File: config/locales/en.yml [hunk 4]
```diff
@@ -13,8 +14,11 @@ en:
   send_me_reset_password_instructions: 'Send me reset password instructions'
   forgot_password: 'Forgot your password?'
   all_transactions: 'All transactions'
+  balance: 'Balance'
   confirm_new_password: 'Confirm new password'
   characters_minimum: ' characters minimum'
   currently_waiting_confirmation_for: 'Currently waiting confirmation for'
   leave_blank_if_you_dont_want_to_change_it: "leave blank if you don\'t want to change it"
```
Comment (AleksSenkou) [inline config/locales/en.yml]: `don't`?
